tests/integration/steps/cassdc_client.go

```diff
@@ -0,0 +1,23 @@
+package steps
+
+import (
+	cassdcapi "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Client returns a controller-runtime client with cass-operator API defined
+func CassDcClient() (client.Client, error) {
```

Comment:
Note that this controller-runtime client be used with other types. Because CassandraDatacenter is a custom resource, you have to tell controller-runtime about it, but it already knows about all the "built-in" types like pods, services, etc.

Reply:
Gotcha. 
I assume this means the client could be used to replace some calls we have in the tests to list pods and services by labels for example?
Ok to defer this to a subsequent issue?

Reply:
>Ok to defer this to a subsequent issue?

Yes, please, thank you 😀